RockPaperScissors/RockPaperScissors/main.swift

```diff
@@ -20,55 +20,85 @@ func printRockPaperScissors() {
  즉, 0이나 4이상의 숫자에 대한 예외처리는 입력 받을 때 처리 되어야할 것 같다.
  code by jake
  */
-func judgmentWinDrawLose(computer : Int, user : Int) {
+func judgmentWinDrawLose(computer : Int, user : Int) -> String {
+    var result : String = ""
     switch computer {
     case 1 : // computer 가위인 경우
-        if user == 1 { print("비겼습니다!") }
-        else if user == 2 { print("이겼습니다!") }
-        else { print("졌습니다!") }
+        if user == 1 { print("비겼습니다!"); result = "비겼습니다!" }
+        else if user == 2 { print("이겼습니다!"); result = "이겼습니다!" }
+        else { print("졌습니다!"); result = "졌습니다!" }
     case 2 : // computer 바위인 경우
-        if user == 1 { print("졌습니다!") }
-        else if user == 2 { print("비겼습니다!") }
-        else { print("이겼습니다!") }
+        if user == 1 { print("졌습니다!"); result = "졌습니다!" }
+        else if user == 2 { print("비겼습니다!"); result = "비겼습니다!" }
+        else { print("이겼습니다!"); result = "이겼습니다!" }
     default : // computer 보인 경우
-        if user == 1 { print("이겼습니다!") }
-        else if user == 2 { print("졌습니다!") }
-        else { print("비겼습니다!") }
+        if user == 1 { print("이겼습니다!"); result = "이겼습니다!" }
+        else if user == 2 { print("졌습니다!"); result = "졌습니다!" }
+        else { print("비겼습니다!"); result = "비겼습니다!" }
     }
+    return result
 }
 
 /*
  컴퓨터의 패를 정해주는 함수
  난수를 통해서 1~3중 하나의 값을 갖는다
  code by kkomal
  */
-@discardableResult
-func isChoiceComputer() -> Int {
-    let computerNum = Int.random(in: 1...3)
-    return computerNum
+func createComputerRockPaperScissors() -> Int {
+    let computerNumber = Int.random(in: 1...3)
+    return computerNumber
 }
 
 /*
- 사용자에게 가위 바위 보 중 하나를 입력받는 함수
- 0~3값만 입력받게하고 다른 값 입력시 예외처리
+ 사용자에게 입력받는 함수
  code by kkomal
  */
-func inputRockPaperScissors() {
+func inputRockPaperScissors() -> String {
+    // 입력값이 nil이 되는 경우 종료시켜준다.
+    guard let input = readLine() else { print("입력값이 nil이라서 종료됩니다."); exit(0) }
```

Comment:
https://wotjd.github.io/2019/06/ios-%EC%95%B1-%EC%A2%85%EB%A3%8C%ED%95%98%EA%B8%B0-how-to-quit-app-in-swift/